lua/dapui/render/loop.lua

```diff
@@ -54,9 +54,22 @@ function M.register_buffer(element_name, buf)
       .. buf
       .. ")"
   )
+  vim.cmd("augroup DAPUIExpandLongLines" .. buf)
+  vim.cmd(
+    "autocmd CursorHold <buffer="
+      .. buf
+      .. "> lua require(\"dapui.render.line_hover\").show_delayed()"
+  )
+  vim.cmd(
+    "autocmd BufLeave,TabClosed <buffer="
+      .. buf
+      .. "> lua require(\"dapui.render.line_hover\").hide_existing_window()"
+  )
+  vim.cmd("augroup END")
 end
 
 function M.remove_buffer(element_name, buf_to_remove)
+  vim.cmd("autocmd! DAPUIExpandLongLines" .. buf_to_remove)
```

Comment:
Lets just add `au!` to the `augroup` definition when created. We don't need to clean up the last one.

Reply:
Since each name has been assigned to the element name, cleaning in standard (as above) way started working.